packages/sfui/frameworks/nuxt/src/module.ts

```diff
@@ -1,7 +1,9 @@
-import { defineNuxtModule, addComponent, addImportsSources, installModule, addImports } from '@nuxt/kit';
-import { NuxtOptions } from '@nuxt/schema';
+import { defineNuxtModule, addComponent, addImportsSources, installModule } from '@nuxt/kit';
+import { type NuxtOptions } from '@nuxt/schema';
 import * as storefrontUi from '@storefront-ui/vue';
 import { tailwindConfig } from '@storefront-ui/vue/tailwind-config';
+import { type RawFile } from 'tailwindcss/types/config';
+import '@nuxtjs/tailwindcss';
```

Comment:
without this import `NuxtOptions['tailwindcss']` does not work and throw error that `tailwindcss` does not exists on `NuxtOptions`, because there is typing file generated on tailwindcss side that `NuxtOptions` contain this property.